### PR TITLE
st7735 Docs

### PR DESCRIPTION
**Pull request by SenexCrenshaw**
## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#1066

## Checklist:

  - [X] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [X] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Review by rradar — APPROVED** [review]

**Comment by glmnet** [comment]
Check titles there is the word 'expander' a few times

File: components/display/st7735.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
+ST7735 I/O Expander
+===================
```
Comment (jesserockz) [inline components/display/st7735.rst]:
```suggestion
ST7735 Display
==============
```

**Review by jesserockz — CHANGES_REQUESTED** [review]
I notice `OLED` a lot in the docs, but the screen is also sometimes LCD...

File: components/display/st7735.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+`Adafruit <https://www.adafruit.com/product/358>`__) in ESPHome.
+It uses the :ref:`SPI Bus <spi>` for communication.
+
+.. figure:: images/st7735.jpg
+    :align: center
+    :width: 75.0%
+
+    ST7735 Display
+
+There are numerous board types out there. Some initialize differntly as well. This driver will take a few options to narrow down the right settings.
```
Comment (jesserockz) [inline components/display/st7735.rst]:
```suggestion
There are numerous board types out there. Some initialize differently as well. This driver will take a few options to narrow down the right settings.
```

**Review by jesserockz — APPROVED** [review]

**Pull request merged**